<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFileDialog</class>
 <widget class="QDialog" name="LogFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log File Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_aliases">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Available Aliases:</string>
   </property>
   <property name="buddy">
    <cstring>txt_dateAliases</cstring>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_dateAliases">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1181</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1181</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_file">
      <property name="text">
       <string>Log File Name:</string>
      </property>
      <property name="buddy">
       <cstring>txt_fileName</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="txt_fileName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_format">
      <property name="text">
       <string>Log File Format Name:</string>
      </property>
      <property name="buddy">
       <cstring>txt_formatName</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="txt_formatName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_regex">
      <property name="text">
       <string>Specify Date Parsing Regex:</string>
      </property>
      <property name="buddy">
       <cstring>lnE_dateRegex</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lnE_dateRegex"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_saveAndExit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>SAVE ALL + CLOSE</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/resources/res/icons/Save.png</normaloff>:/resources/res/icons/Save.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lnE_dateRegex</tabstop>
  <tabstop>txt_dateAliases</tabstop>
  <tabstop>txt_fileName</tabstop>
  <tabstop>txt_formatName</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
